rules/windows/credential_access_cmdline_dump_tool.toml

```diff
@@ -26,8 +27,8 @@ query = '''
 process where event.type in ("start", "process_started") and
 /* update here with any new lolbas with dump capability */
 (process.pe.original_file_name == "procdump" and process.args : "-ma") or
-(process.name : "ProcessDump.exe" and not process.parent.executable : "C:\\Program Files*\\Cisco Systems\\*.exe") or
-(process.pe.original_file_name == "WriteMiniDump.exe" and not process.parent.executable : "C:\\Program Files*\\Steam\\*.exe") or
+(process.name : "ProcessDump.exe" and not process.parent.executable regex~ """C:\\Program Files( \(x86\))?\\Cisco Systems\\*""") or
```

Comment:
remind me, would switching to literals `"""` mean we need to convert to single `\` for the path

Reply:
we're not escaping `\` here for the string literal, but for the regular expression (since `\` is special)

Reply:
I understand the use of `\` in regex. I am asking if we _can_ use single escapes with `"""` for paths, because the regex makes it a bit harder to read and single would alleviate that a bit